projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/CreateJob.tsx

```diff
@@ -0,0 +1,148 @@
+                    </div>
+                </div>
+                <div className='jp-vdk-input-wrapper'>
+                    <label className='jp-vdk-label' htmlFor="jobName">Job Name:</label>
+                    <input type="text" id="jobName" className='jp-vdk-input' placeholder={this.props.jobName} onChange={this._onNameChange} />
+                </div>
+                <div className='jp-vdk-input-wrapper'>
+                    <label className='jp-vdk-label' htmlFor="jobTeam">Job Team:</label>
+                    <input type="text" id="jobTeam" className='jp-vdk-input' placeholder="jobs" onChange={this._onTeamChange} />
+                </div>
+                <div className='jp-vdk-input-wrapper hidden'>
+                    <label className='jp-vdk-label' htmlFor="jobPath">Path to parent directory:</label>
+                    <input type="text" id="jobPath" className='jp-vdk-input' placeholder={this.props.jobPath} onChange={this._onPathChange} />
+                </div>
+                <div className='jp-vdk-input-wrapper hidden'>
+                    <label className='jp-vdk-label' htmlFor="jobRestApiUrl">Rest Api URL:</label>
+                    <input type="text" id="jobRestApiUrl" className='jp-vdk-input' placeholder="rest-api-url" onChange={this._onRestApiUrlChange} />
+                </div>
+            </>
+        );
+    }
+    /**
+   * Callback invoked upon choosing local checkbox
+   */
+    private _onLocalClick() {
+        return (event: React.MouseEvent) => {
+            this._handleCheckboxEvent("Local", "jobPath", "local")
+        }
+    }
+    /**
+  * Callback invoked upon choosing cloud checkbox
+  */
+    private _onCloudClick() {
+        return (event: React.MouseEvent) => {
+            this._handleCheckboxEvent("Cloud", "jobRestApiUrl", "cloud")
+        }
+    }
+    /**
+    * Helper method for checkbox click events which include removing/adding visibility of input element to the dialog
+    */
+   private _handleCheckboxEvent(checkboxId: string, inputId: string, createJobType: string){
+    let checkbox = document.getElementById(checkboxId);
+    let input = document.getElementById(inputId);
+    if (checkbox?.classList.contains("checked")) {
+        checkbox.classList.remove("checked");
+        input?.parentElement?.classList.add("hidden");
+        localStorage.removeItem(createJobType);
+    }
+    else {
+        checkbox?.classList.add("checked");
+        localStorage.setItem(createJobType, "1");
+        input?.parentElement?.classList.remove("hidden");
+    }
+   }
+    /**
+   * Callback invoked upon changing the job name input.
+   *
+   * @param event - event object
+   */
+    private _onNameChange = (event: any): void => {
+        const nameInput = event.currentTarget as HTMLInputElement;
+        let value = nameInput.value;
+        if (!value) {
+            value = this.props.jobPath;
+        }
+        sessionStorage.setItem("job-name", value);
```

Comment:
we should keep track (documentation) on what data is session-stored by design;

Reply:
Since I predict more uses of session storage a added an issue (https://github.com/vmware/versatile-data-kit/issues/1484_ for this problem